src/components/Swatches.jsx

```diff
@@ -1,11 +1,25 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-const Swatches = ({ items = [], children }) => (
+const Swatches = ({ items = [], aliasesKey, children }) => (
 	<>
 		{Array.isArray(items)
-			? items.map((value, index) => children(index, value))
-			: Object.keys(items).map(key => children(key, items[key]))}
+			? items.map((value, index) =>
+					children(
+						index,
+						value,
+						aliasesKey && items[aliasesKey] ? items[aliasesKey][index] : null
+					)
+			  )
+			: Object.keys(items)
+					.filter(key => key !== aliasesKey)
+					.map(key =>
+						children(
+							key,
+							items[key],
+							aliasesKey && items[aliasesKey] ? items[aliasesKey][key] : null
+						)
+					)}
```

Comment:
TBH I'm not sure I want to accept this code into the repo. Maybe the best course of action would be to do this in your repo after all. You can compose things with `Swatches` and `Swatch`, add the resolution logic on top of what'e being done and extend provided primitives. 

Thoughts?

Reply:
Maybe we can just use this repo to create a custom library? The only component which can be reused without any change is `Swatch` (which gives copy-to-clipboard feature)

Reply:
I'm not sure yet. I'd like to support aliases but I'm not sure the code complexity is worth it. Is there a simpler way of getting aliases for tokens? Ideally, as the user, I should not care about providing alias keys to the library. Let me poke with it a bit and let me know.